### DISCUSSION TITLE
When copilot guesses my comments...

### DISCUSSION THREAD

**Discussion by ropel** [community/community#13950] [Copilot Conversations]
I'm trying copilot and its ability is impressive, but a question comes into my mind: if it's able to guess what I'm going to write in a comment, is it worth writing ? I think that comments should explain what is otherwise difficult to understand, not what is clearly visible looking at the code.

**Comment by steinhh** [comment]
Hear hear!! Also, I'd really like to be able to turn off comments altogether at the end of a line, after a semi-colon (e.g., in javascript). If you need a comment to explain what a single line does, you're not doing it right! And if you don't need a comment, there shouldn't be one. Does anyone know about an existing feature request for turning off end-of-line comments altogether? If not, I'll make one :)
